doc/source/index.rst

```diff
@@ -27,9 +27,10 @@ discussions, release procedure and more.
 Authors and Contributors
 ========================
 
-The main authors of NiBabel are `Matthew Brett`_, `Michael Hanke`_ and `Stephan
-Gerhard`_.  The authors are grateful to the following people who have
-contributed code and discussion (in rough order of appearance):
+The main authors of NiBabel are `Matthew Brett`_, `Michael Hanke`_, `Ben
+Cipollini`_, Marc-Alexandre Côté, Chris Markiewicz, `Stephan Gerhard`_ and Eric
+Larson.  The authors are grateful to the following people who have contributed
```

Comment:
Thanks for that - added.

@effigies - have you got a favored home page we can point to?